kuzzleio/documentation-V2
# Spring clean-up of index.js containing all the metalsmith process

## What does this PR do?
* refactor in an other file handlebars helpers and metalsmith build args options
* clean unused packages 
* clean metalsmith build process

### How should this be manually tested?
https://deploy-preview-37--kuzzle-doc-v2.netlify.com/sdk-reference/
Navigate and check everything is working (except the language-selector, it's in another PR)
